wrappers/kn/src/nativeMain/kotlin/zxingcpp/ImageView.kt

```diff
@@ -0,0 +1,64 @@
+package zxingcpp
+
+import cnames.structs.zxing_ImageView
+import kotlinx.cinterop.CPointer
+import kotlinx.cinterop.ExperimentalForeignApi
+import kotlinx.cinterop.addressOf
+import kotlinx.cinterop.pin
+import zxingcpp.cinterop.*
+
+@OptIn(ExperimentalForeignApi::class)
+data class ImageView(
+	val data: UByteArray,
+	val width: Int,
+	val height: Int,
+	val format: ImageFormat,
+	val rowStride: Int = 0,
+	val pixStride: Int = 0,
+) {
+
+	@OptIn(ExperimentalStdlibApi::class)
+	internal class ClosableCImageView(
```

Comment:
Looking at this, I wondered why you had to introduce this here but not for the `ReaderOptions`? Maybe because I was talking about a `data class` earlier? Sorry if I used that term in the wrong way. I wanted to express the fact that I don't see value in having a class hierarchy  here. But looking at this now makes it look like this should be implemented just like `ReaderOptions` with regards to the con- and de-struction.

Reply:
Sorry, I was wrongly regarded constructor parameters as `var`, regarding the value of `zxing_ImageView` are not changable, I decided to create a c object everytime its getter was called... 

Fix this latter.

Reply:
fixed in: https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/58489d219243f1f156bb42da90d5d2764ee70aa9